libs/dh/admin/feature-user-roles/src/lib/dh-user-roles.component.ts

```diff
@@ -27,21 +34,24 @@ import { WattEmptyStateComponent } from '@energinet-datahub/watt/empty-state';
 import { WattTableColumnDef, WATT_TABLE } from '@energinet-datahub/watt/table';
 import { WATT_EXPANDABLE_CARD_COMPONENTS } from '@energinet-datahub/watt/expandable-card';
 
-import { DhUser, UpdateUserRoles } from '@energinet-datahub/dh/admin/shared';
 import {
-  GetUserRoleViewDocument,
-  UserRoleViewDto,
-} from '@energinet-datahub/dh/shared/domain/graphql';
-import { lazyQuery } from '@energinet-datahub/dh/shared/util-apollo';
+  DhActorUserRole,
+  DhActorUserRoles,
+  DhUser,
+  UpdateUserRoles,
+} from '@energinet-datahub/dh/admin/shared';
 
 import {
   FilterUserRolesPipe,
   UserRolesIntoTablePipe,
 } from './dh-filter-user-roles-into-table.pipe';
+import { WattIconComponent } from '@energinet-datahub/watt/icon';
+import { WattTooltipDirective } from '@energinet-datahub/watt/tooltip';
 
 @Component({
   selector: 'dh-user-roles',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
```

Comment:
suggestion: now that encapsulation is turned off, let's move the styles under the component's selector to avoid leaking.